src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -3765,3 +3765,41 @@ class AccessStatementSegment(BaseSegment):
             ),
         ),
     )
+
+
+class CreateTypeStatementSegment(BaseSegment):
+    """A `CREATE TYPE` statement.
+
+    This is based around the Postgres syntax.
+    https://www.postgresql.org/docs/current/sql-createtype.html
+
+    Note: This is relatively permissive currently
+    and does not lint the syntax strictly, to allow
+    for some deviation between dialects.
```

Comment:
Please update this comment -- looks like it was copied from the ANSI definition.

Reply:
Argh, sorry, I totally missed that